### PR TITLE
chore: Remove unused feature flags

### PR DESCRIPTION
**Pull request by markstory**
These feature flags had no checks in either sentry or getsentry and are just adding overhead to Organization Details response times.

I'll remove these flags from Flagr as well. Tagging DnD as many of these features were there and I updated one of their acceptance tests.

### CODE REVIEW

File: src/sentry/conf/server.py [hunk 1]
```diff
@@ -1067,8 +1031,6 @@ def create_partitioned_queues(name):
     "organizations:invite-members": True,
     # Enable rate limits for inviting members.
     "organizations:invite-members-rate-limits": True,
-    # Enable removing issue from issue list if action taken.
-    "organizations:issue-list-removal-action": False,
```
Comment (wedamija) [inline src/sentry/conf/server.py:1071]: @scttcper Was this a feature we were planning on adding soon? Wasn't sure if it was related to that pr from last week?

**Review by wedamija — APPROVED** [review]
Nice cleanup!

**Review by wmak — APPROVED** [review]

File: src/sentry/conf/server.py (hunk 1 above)
Comment (scttcper) [reply src/sentry/conf/server.py:1071]: good catch, this is a flag we're about to use

File: src/sentry/conf/server.py [hunk 2]
```diff
@@ -1114,20 +1072,10 @@ def create_partitioned_queues(name):
...
     # Enable the new images loaded design and features
     "organizations:images-loaded-v2": True,
-    # Enable the mobile screenshots feature
-    "organizations:mobile-screenshots": False,
-    # Store release bundles as zip files instead of single files
-    "organizations:release-archives": False,
-    # Enable the release details performance section
-    "organizations:release-comparison-performance": False,
-    # Enable percent displays in issue stream
-    "organizations:issue-percent-display": False,
```
Comment (scttcper) [inline src/sentry/conf/server.py:1130]: i think this feature still exists

File: src/sentry/conf/server.py [hunk 3]
```diff
@@ -1043,22 +1017,12 @@ def create_partitioned_queues(name):
...
     # Enable readonly dashboards
     "organizations:dashboards-basic": True,
     # Enable custom editable dashboards
     "organizations:dashboards-edit": True,
-    # Enable dashboard widget library
-    "organizations:widget-library": False,
     # Enable metrics in dashboards
     "organizations:dashboards-metrics": False,
-    # Enable issue widgets in dashboards
-    "organizations:issues-in-dashboards": False,
```
Comment (scttcper) [inline src/sentry/conf/server.py:1057]: seeing this flag in code

File: src/sentry/conf/server.py (hunk 2 above)
Comment (markstory) [reply src/sentry/conf/server.py:1130]: If missed this one. I'll double check all the removals for the unprefixed names too.

File: src/sentry/conf/server.py [hunk 4]
```diff
@@ -1014,10 +992,6 @@ def create_partitioned_queues(name):
     "organizations:release-health-check-metrics": False,
     # True if differences between the metrics and sessions backend should be reported
     "organizations:release-health-check-metrics-report": False,
-    # Enable metric aggregate in metric alert rule builder
-    "organizations:metric-alert-builder-aggregate": False,
-    # Enable threshold period in metric alert rule builder
-    "organizations:metric-alert-threshold-period": False,
```
Comment (scttcper) [inline src/sentry/conf/server.py:1020]: `metric-alert-threshold-period` still exists

**Review by dashed — APPROVED** [review]

**Pull request merged**